tests/system/src/test/java/test/robot/javafx/scene/tableview/TableViewClickOnTroughTest.java

```diff
@@ -0,0 +1,202 @@
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.geometry.Bounds;
+import javafx.scene.Scene;
+import javafx.scene.control.ScrollBar;
+import javafx.scene.control.TableCell;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableRow;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.PropertyValueFactory;
+import javafx.scene.input.MouseButton;
+import javafx.scene.layout.StackPane;
+import javafx.scene.robot.Robot;
+import javafx.scene.text.Font;
+import javafx.stage.Stage;
+import javafx.stage.StageStyle;
+import javafx.stage.WindowEvent;
+import javafx.util.Callback;
+import test.util.Util;
+
+/*
+ * [openjdk/jfx] 8173321: Click on trough has no effect when cell height > viewport (PR #985):
+ * Test to verify
+ * - click on trough moves the Scrollbar even if the table cell's height is greater than the table's height
+ * - size of ScrollBar is greater than minimum size even if the table cell's height is greater than the table's height
+ */
+public class TableViewClickOnTroughTest {
+
+    static Robot robot;
+    static volatile Stage stage;
+    static volatile Scene scene;
+    static final int SCENE_WIDTH = 800;
+    static final int SCENE_HEIGHT = 250;
+    static CountDownLatch startupLatch = new CountDownLatch(1);
+
+    private static TableView<TableEntry> table;
+    public static TableRow<TableEntry> tableRow;
+
+    public static void main(String[] args) {
+        TableViewClickOnTroughTest test = new TableViewClickOnTroughTest();
+        test.moveTroughTest();
+    }
+
+    @Test
+    public void moveTroughTest() {
+        ScrollBar verticalBar = (ScrollBar) table.lookup(".scroll-bar:vertical");
+        StackPane thumb = (StackPane) verticalBar.getChildrenUnmodifiable().stream()
+                .filter(c -> c.getStyleClass().contains("thumb")).findFirst().orElse(null);
+        StackPane trackBackground = (StackPane) verticalBar.getChildrenUnmodifiable().stream()
+                .filter(c -> c.getStyleClass().contains("track-background")).findFirst().orElse(null);
+        Bounds verticalBarBoundsInScreen = verticalBar.localToScreen(verticalBar.getBoundsInLocal());
+        Bounds thumbBoundsInScreen = thumb.localToScreen(thumb.getBoundsInLocal());
+        double posX = verticalBarBoundsInScreen.getCenterX();
+        // set posY to point into the middle of the area of the verticalBar under the thumb.
+        double posY = verticalBarBoundsInScreen.getMaxY() - ((verticalBarBoundsInScreen.getMaxY()-thumbBoundsInScreen.getMaxY())/2.0);
+        double oldPosition = verticalBar.getValue();
+
+        // Wait a second for the bar to be moved
+        try {
+            Thread.sleep(1000);
+        } catch (InterruptedException e1) {
+            e1.printStackTrace();
+        }
```

Comment:
You can change this to `Util.sleep(1000);` and eliminate the try/catch here and below.